src/components/Navbar.js

```diff
@@ -2,16 +2,14 @@ import { Link } from 'react-router-dom';
 
 function Navbar() {
   return (
-    <div>
-      <h1>Bookstore</h1>
-      <nav>
-        <Link to="/">Books</Link>
-        {' '}
-        {' '}
-        |
-        {' '}
-        <Link to="/categories">Categories</Link>
-      </nav>
+    <div className="nav-container">
+      <div className="navbar">
+        <h1 className="Bookstore-CMS">Bookstore CMS</h1>
+        <nav>
+          <Link to="/" className="BOOKS">Books</Link>
+          <Link to="/categories" className="CATEGORIES">Categories</Link>
+        </nav>
+      </div>
```

Comment:
- [ ] The design should be the same. Kindly update it. [Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md#:~:text=Your%20final%20design%20should%20match%20Zeplin%27s%20design.)

![image](https://user-images.githubusercontent.com/38635158/181615767-0058cb38-0b6b-4941-9c0a-372fe61261cb.png)

___
### ORIGINAL DESIGN
![image](https://user-images.githubusercontent.com/38635158/181615613-8477b6c4-5658-4af7-bdf9-d2e79bcbd17e.png)